frontend/src/app/course/courseCard.jsx

```diff
@@ -0,0 +1,20 @@
+// @flow
+import * as React from 'react';
+import Card from 'react-bootstrap/Card';
+
+import './styles/courseCard.scss';
+import type { Course } from './types';
+
+const CourseCard = (course: Course): React.Node => {
+    return (
+        <Card className={'card shadow-sm show-pointer'}>
+            <Card.Body>
+                <Card.Title>{course.name}</Card.Title>
+                <hr />
```

Comment:
I think this should not be used. If you want you can use the Card.Header (see the doc)